class.jetpack.php

```diff
@@ -1302,8 +1302,8 @@ public static function is_development_mode() {
 			$development_mode = JETPACK_DEV_DEBUG;
 		}
 
-		elseif ( site_url() && false === strpos( site_url(), '.' ) ) {
-			$development_mode = true;
+		elseif ( $site_url = site_url() ) {
+		        $development_mode = false === strpos( $site_url, '.' );
```

Comment:
Can you fix the indentation here? It was changed from a tab to spaces (and more than needed). The spacing before the `elseif` could also be removed.